packages/commonwealth/server/routing/router.ts

```diff
@@ -300,7 +300,6 @@ function setupRouter(
   router.post(
     '/updateThreadStage',
     passport.authenticate('jwt', { session: false }),
-    databaseValidationService.validateAuthor,
```

Comment:
Why did we remove this? Don't we need to restrict who can change the thread stage?

Reply:
There was no author validation before. I think I added this by accident during the validateAuthor branch. For ex. https://github.com/hicommonwealth/commonwealth/blob/d07099aea31d9c9addfe50f00b15979a6cc2c99b/packages/commonwealth/server/routes/updateThreadStage.ts